extensions/wikia/AdEngine/js/video/player/porvata/porvata.js

```diff
@@ -28,18 +33,37 @@ define('ext.wikia.adEngine.video.player.porvata', [
 			}).then(function (video) {
 				log(['porvata video player created', video], log.levels.debug, logGroup);
 
+				function inViewportCallback(isVisible) {
+					// Play video automatically only for the first time
+					if (isVisible && !autoPlayed && params.autoPlay) {
+						video.play();
+						autoPlayed = true;
+					// Don't resume when video was paused manually
+					} else if (isVisible && shouldResume) {
```

Comment:
I know, either way please rename so it reflects what it represents:
I'm fine with either - `pausedByUser` (with changed conditions) or `pausedAutomatically`. However, I lean toward `pausedByUser` because it's more descriptive.